Contact Us page and fixed Problem of footer

Rajkotadiya opened the pull request:
Fixed #11 #41 
Contact us page is created and problem of footer edited.
## Mocks
![image](https://user-images.githubusercontent.com/46588494/96092241-3dfb1a00-0ee8-11eb-8be0-c3fdf913ad23.png)
![image](https://user-images.githubusercontent.com/46588494/96092227-389dcf80-0ee8-11eb-9511-b20f804a8b01.png)

keshakaneria commented:
Looks great! Just add the screenshots of how does the page look in the description and we are good to go to merge this PR 🥳

keshakaneria commented:
This looks great! Now as we want is on left side there needs to be a div where there are links given of all the social media tasks and on right side, this map shall be attached. Since the above 4 boxes of different social medias will contain the scrollable posts, we are putting them here for just a dummy pic

keshakaneria commented:
Also add the background picture from [here](https://drive.google.com/drive/folders/108bwRPAhJ1SCQmE-3VIB2mHnmQS_XRYO)

Rajkotadiya commented:
what i have put in that 4 boxes ?

keshakaneria commented:
Something sort of this
![image](https://user-images.githubusercontent.com/46588494/96104215-2f1b6400-0ef6-11eb-9e35-cf7f0a0a8284.png)

keshakaneria commented:
Do pull the changes before editing the requested changes @Rajkotadiya

keshakaneria reviewed (changes requested):
- add description with images
- put the links on left side and map on right side

Rajkotadiya commented:
i have added links on the left and i don't get what is actually conflicting in contactUs.html and imgs/contactus.jpg.

so I have just changed name of contactus.jpg

keshakaneria reviewed (approved):
LGTM! 😄
